packages/react/src/components/ValueCard/valueCardUtils.js

```diff
@@ -66,9 +67,12 @@ export const determineMaxValueCardAttributeCount = (size, currentAttributeCount)
       break;
     case CARD_SIZES.MEDIUMTHIN:
     case CARD_SIZES.MEDIUM:
-    case CARD_SIZES.MEDIUMWIDE:
       attributeCount = 3;
       break;
+    case CARD_SIZES.SMALLFULL:
+    case CARD_SIZES.MEDIUMWIDE:
```

Comment:
This works but this function isn't used at all by ValueCard anymore. Its simply here until we can remove it in the next release